includes/Support/WordPress.php

```diff
@@ -118,4 +118,27 @@ public static function clearScheduledHook(string $hook): void
     {
         \wp_clear_scheduled_hook($hook);
     }
+
+    /**
+     * Returns all meta field keys for the given post type
+     *
+     * @param string $postType The post type to get the meta keys for.
+     * @return array
+     */
+    public static function getAllMetaKeysForPostType(string $postType): array
+    {
+        global $wpdb;
+
+        $query = "
+            SELECT DISTINCT($wpdb->postmeta.meta_key) 
+            FROM $wpdb->posts 
+            LEFT JOIN $wpdb->postmeta 
+            ON $wpdb->posts.ID = $wpdb->postmeta.post_id 
+            WHERE $wpdb->posts.post_type = '%s'
+            # exclude hidden meta keys
```

Comment:
if this is sql the # seems not to fit in there properly

Reply:
It is SQL and that line is a comment.
https://dev.mysql.com/doc/refman/8.0/en/comments.html